specs/discoverable_partitions_specification.md

```diff
@@ -185,19 +185,42 @@ this specification.
 [systemd-id128(1)](https://www.freedesktop.org/software/systemd/man/systemd-id128.html)'s
 `show` command may be used to list those GPT partition type UUIDs.
 
-## Partition Names
+## Partition Ownership
 
-For partitions of the types listed above it is recommended to use
-human-friendly, descriptive partition names in the GPT partition table, for
-example "*Home*", "*Server* *Data*", "*Fedora* *Root*" and similar, possibly
-localized.
+If two Linux-based operating systems are installed on the same disk, the scheme
+above suggests that they may share the swap, `/home/`, `/srv/`, `/var/tmp/`,
+ESP, XBOOTLDR. However, they should each have their own root, `/usr/` and
+`/var/` partition.
```

Comment:
This is phrased unclear regarding whether or not swap/home/srv/var-tmp are actually allowed to be shared or not, and whether partition ownership applies to them.